connectors/src/connectors/confluence/temporal/activities.ts

```diff
@@ -252,7 +253,7 @@ export async function markPageHasVisited({
 interface ConfluenceUpsertPageInput {
   page: NonNullable<Awaited<ReturnType<ConfluenceClient["getPageById"]>>>;
   spaceName: string;
-  parents: string[];
+  parents: [string, ...string[], string];
```

Comment:
This one is weird 🙃

Reply:
What if I have only one?

Reply:
you can't, i'm trying to enforce a Confluence logic rule that there's always gonna be the internalId and the spaceID (or the hidden node thing if parents are not already computed)

Reply:
Ah right it's confluence 👍 Thought it was datasource.ts

Reply:
hehe